internal/pkg/secret/encryption_other.go

```diff
@@ -91,7 +92,7 @@ func Decrypt(username, encrypted string, salt, nonce []byte) (string, error) {
 	}
 	decryptedPassword, err := aesgcm.Open(nil, nonce, cipherText, []byte(username))
 	if err != nil {
-		return "", err
+		return "", errors.New(fmt.Sprintf("%s. %s", err.Error(), "Check cli read/write permission to /etc/machine-id."))
```

Comment:
```suggestion
		return "", fmt.Errorf("CLI does not have write permission for `/etc/machine-id`: %w", err)
```

Reply:
I've used a few command line tools that will prompt the user for their system password when they run into permission issues like this one. Do you think we could try something similar here?

Reply:
I'm not sure this is the only root cause... it could be just that `open` failed

Reply:
Gotcha, let's just suggest that they manually add write permissions to the file.

Reply:
I was hoping there might be a solution that doesn't require user intervention, since we've already gotten multiple customer complaints.